### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kudrat89**
pervii comit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # github_pt
 GitHub parsing test
+первый комит в 19:54 23,06,2023
```
Comment (aigulok) [inline README.md:3]: This is a test comment

Comment (aigulok) [inline README.md:3]: this is another test comment

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # github_pt
 GitHub parsing test
```
Comment (aigulok) [inline README.md:2]: hello

Comment (Kudrat89) [reply README.md:2]: 11

**Comment by Kudrat89** [comment]
tes

**Pull request merged**